### PR TITLE
DOCS-2425: Merge autogen: mlmodel, motion

### PR DESCRIPTION
**Pull request by andf-viam**
Merge autogen content for `mlmodel`, `motion`.
- Includes latest Go code snippets from https://github.com/viamrobotics/rdk/pull/3970
- Noticed get_pose uses split code sample approach, first time in PySDK this approach is used. Created PR to fix (https://github.com/viamrobotics/viam-python-sdk/pull/675) and re-ran with staged version to get fixed version in this PR.

### CODE REVIEW

File: docs/services/motion/_index.md [hunk 1]
```diff
@@ -73,840 +73,7 @@ For examples showing how to construct the `ResourceName`, see the code samples b
...
-
-- `extra` (Mapping[str, Any]) (_optional_): A generic struct, containing extra options to pass to the underlying RPC call.
-
-**Returns:**
-
-- [(bool)](https://docs.python.org/3/library/stdtypes.html#bltin-boolean-values): Whether the move was successful.
-
-For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/services/motion/index.html#viam.services.motion.MotionClient.move).
-
-```python {class="line-numbers linkable-line-numbers"}
```
Comment (npentrel) [inline docs/services/motion/_index.md:136]: This is a much better example I think?

File: docs/services/motion/_index.md [hunk 2]
```diff
@@ -73,840 +73,7 @@ For examples showing how to construct the `ResourceName`, see the code samples b
...
-# Assume that the connect function is written and will return a valid machine.
-robot = await connect()
-
-motion = MotionClient.from_robot(robot=robot, name="builtin")
-gripperName = Gripper.get_resource_name("my_gripper")
-gripperPoseInWorld = await motion.get_pose(component_name=gripperName,
-                                           destination_frame="world")
-```
-
-For a more complicated example, take the same scenario and get the pose of the same gripper with respect to an object situated at a location (100, 200, 0) relative to the "world" frame:
```
Comment (npentrel) [inline docs/services/motion/_index.md:272]: This feels like it should also remain - added as a secondary example to the python docs

File: static/include/services/apis/overrides/methods/go.motion.MoveOnMap.req.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<!-- preserve-formatting -->
```
Comment (npentrel) [inline static/include/services/apis/overrides/methods/go.motion.MoveOnMap.req.md:1]: This type of override makes me very nervous. It'll be outdated so quickly. But we'll address it later

**Review by npentrel — APPROVED** [review]
Lgtm but there are two longer choose snippets getting lost. Can you confirm that's intended or open a ticket to add them to upstream?

File: docs/services/motion/_index.md (hunk 2 above)
Comment (andf-viam) [reply docs/services/motion/_index.md:272]: Sure, re-added it back 👍

File: docs/services/motion/_index.md (hunk 1 above)
Comment (andf-viam) [reply docs/services/motion/_index.md:136]: Ah, another split code sample upstream: https://python.viam.dev/autoapi/viam/services/motion/client/index.html#viam.services.motion.client.MotionClient.move

I'll make a PR to fix and re-run with the fixed,staged version. 👍

**Pull request merged**